StigData/Archive/Windows.Server.2016/U_MS_Windows_Server_2016_DC_STIG_V1R9_Manual-xccdf.log

```diff
@@ -11,3 +11,4 @@ V-73591::Registry Path: \SOFTWARE\ Policies\Microsoft\Windows\PowerShell\ScriptB
 V-73551::Value: 0x00000000 (0) (Security), 0x00000001 (1) (Basic)::Value: 0x00000000 (0) (Security) or 0x00000001 (1) (Basic)
 V-73711::Value: 0x00000002 (2) (Prompt for consent on the secure desktop)::Value: 1 or 2
 V-73755::Passwords for application accounts with this user right must be protected as highly privileged accounts.::""
+V-73375::*::''
```

Comment:
Thanks nehru. It looks like V-73369 has the same values as the other STIG referenced. Is there a particular value we know v-73375 needs to avoid the failure?